platform/net/firewall_provider_linux.go

```diff
@@ -89,6 +100,18 @@ func SetupNatsFirewall(mbus string) error {
 			return bosherr.WrapError(err, "Iptables Error inserting for monit rule")
 		}
 	}
+
+	// For nats iptables rules we default to V4 unless below dns resolution gives us a V6 target
+	ipVersion := iptables.ProtocolIPv4
+	// Check if we're dealing with a V4 Target
```

Comment:
This comment is wrong or? We are checking wether we deal with IP v6 here.

Reply:
yeah we're checking if it's a V6 Address by trying to create a V4. If that fails, we have a V6. Since the check is actually trying to run To4() I thought it's less confusing.